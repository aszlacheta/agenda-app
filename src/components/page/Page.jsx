import React from 'react';
import Footer from '../footer/Footer';
import Header from '../header/Header';
import './Page.scss';

export default function Page({ children }) {

    return <>
        <Header />
        <div className="page-children-container">
            {children}
        </div>
        <Footer />
    </>
}
