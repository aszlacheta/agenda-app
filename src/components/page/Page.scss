@use '../variables.scss';

.page-children-container {
    display: flex;
    flex: 1;

    margin: variables.$pageMargin;
}