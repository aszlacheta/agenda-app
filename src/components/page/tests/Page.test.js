import React from 'react'
import renderer from 'react-test-renderer'
import Page from '../Page'

jest.mock('../../header/Header')
jest.mock('../../footer/Footer')

describe('Page', () => {
  it('check if renders properly', () => {
    const tree = renderer
      .create(<Page />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
