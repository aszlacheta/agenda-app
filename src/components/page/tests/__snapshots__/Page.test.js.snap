// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page check if renders properly 1`] = `
Array [
  <div
    className="mocked-header"
  />,
  <div
    className="page-children-container"
  />,
  <div
    className="mocked-footer"
  />,
]
`;
